packages/jsii-rosetta/bin/jsii-rosetta.ts

```diff
@@ -69,37 +70,31 @@ function main() {
       '(EXPERIMENTAL) mutates one or more assemblies by adding documentation examples to top-level types',
       (command) =>
         command
-          .positional('TABLET', {
-            type: 'string',
-            required: true,
-            describe: 'Language tablet to read',
-          })
           .positional('ASSEMBLY', {
             type: 'string',
             string: true,
             default: new Array<string>(),
             describe: 'Assembly or directory to mutate',
           })
-          .option('log', {
+          .option('log-file', {
             alias: 'l',
-            type: 'boolean',
-            describe: 'Test all algorithms and log results to an html file',
-            default: false,
-          })
-          .option('output', {
-            alias: 'o',
             type: 'string',
             describe: 'Output file to store logging results. Ignored if -log is not true',
             default: DEFAULT_INFUSION_RESULTS_NAME,
           })
-          .demandOption('TABLET'),
+          .option('cache-to', {
+            alias: 'o',
```

Comment:
I don't think we can have two-letter aliases. Doesn't fit GNU commandline flag style.

Since this is a replacement for the old `--output` flag, `-o` seemed okay.